server/src/test/java/io/crate/planner/SelectPlannerTest.java

```diff
@@ -867,11 +867,13 @@ public void testGlobalAggregateOn3TableJoinWithNoMatch() throws Exception {
     @Test
     public void testFilterOnPKSubsetResultsInPKLookupPlanIfTheOtherPKPartIsGenerated() throws Exception {
         SQLExecutor e = SQLExecutor.builder(clusterService, 2, RandomizedTest.getRandom(), List.of())
-            .addTable(
+            .addPartitionedTable(
                 "create table t_pk_part_generated (" +
                 "   ts timestamp with time zone," +
                 "   p as date_trunc('day', ts)," +
-                "   primary key (ts, p))")
+                "   primary key (ts, p)) " +
+                "partitioned by (p)",
+            "p")
             .build();
```

Comment:
Why the change here?

Reply:
no reason, I was confused with the test title, and thought it wanted to check partitioned table with generated column, reverted.